VideoWeb/VideoWeb/ClientApp/src/app/security/authentication/security-service-provider.service.ts

```diff
@@ -0,0 +1,51 @@
+import { Injectable } from '@angular/core';
+import { OidcSecurityService } from 'angular-auth-oidc-client';
+import { SecurityConfigSetupService } from '../security-config-setup.service';
+import { IdpProviders } from '../idp-providers';
+import { MagicLinkSecurityService } from './magic-link-security.service';
+import { ISecurityService } from './security-service.interface';
+import { BehaviorSubject, Observable } from 'rxjs';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class SecurityServiceProviderService {
```

Comment:
How many times can we fit the word Service in a filename??

Reply:
Renamed to SecurityProviderService